config/environments/test.rb

```diff
@@ -1,4 +1,6 @@
 Curves::Application.configure do
+  config.action_mailer.default_url_options = { :host => 'localhost:3030'}
```

Comment:
This step was missing from the devise install instructions long ago.  You need to do this, otherwise it will give the following error:

```
  1) Error:
CommentTest#test_Comment_rating_test:
ActionView::Template::Error: Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true
    app/views/notification_mailer/comment_added.html.erb:5:in `_app_views_notification_mailer_comment_added_html_erb___548536781__589246738'
    app/mailers/notification_mailer.rb:7:in `comment_added'
    app/models/comment.rb:19:in `send_comment_email'
    test/models/comment_test.rb:10:in `block in <class:CommentTest>'

1 tests, 0 assertions, 0 failures, 1 errors, 0 skips
```